kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx

```diff
@@ -73,6 +67,32 @@ const SendMessage: React.FC = () => {
     );
   }, [messageSchema, schemaIsFetched]);
 
+  const {
+    register,
+    handleSubmit,
+    formState: { isSubmitting, isDirty },
+    control,
+    reset,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: useMemo(
```

Comment:
Is there a point in `useMemo` in `defaultValues`?

I'm not sure if it's needed

Reply:
bcs we should monitor changes in keyDefaultValue, contentDefaultValue

Reply:
> The defaultValues for inputs are used as the initial value when a component is first rendered, before a user interacts with it. It is encouraged that you set defaultValues for all inputs to non-undefined values such as the empty string or null.

> defaultValues are cached on the first render within the custom hook. If you want to [reset](https://react-hook-form.com/api/useform/reset) the defaultValues, you should use the reset api.

I see no need in useMemo here